apigee_edge_ui.module

```diff
@@ -168,3 +173,34 @@ function apigee_edge_ui_form_alter(array &$form, FormStateInterface $form_state,
     }
   }
 }
+
+/**
+ * Implements template_preprocess_app_credential_product_list().
+ */
+function apigee_edge_ui_preprocess_app_credential_product_list(array &$variables): void {
+  if (sizeof($variables['content']) <= 5) {
+    return;
+  }
+  $variables['#attached']['library'][] = 'apigee_edge_ui/apigee_edge_ui.app_credential_product_list';
+  $variables['hidden_list'] = [];
+  $variables['hidden_list']['num_of_visible_rows'] = 5;
+  $variables['hidden_list']['attributes'] = new Attribute([
+    'id' => 'app-cred-prod-hidden-list',
+    'class' => ['app-credential-product-list--hidden'],
+  ]);
+  $variables['buttons'] = [];
+  $variables['buttons']['show_more'] = [
+    'text' => t('Show more'),
+    'attributes' => new Attribute([
+      'id' => 'app-cred-prod-list-show-more',
+      'class' => [],
+    ]),
+  ];
+  $variables['buttons']['show_less'] = [
+    'text' => t('Show less'),
```

Comment:
I think it would be better to create the text in Twig with `t` filter and creating the attributes in the Twig as well. This way developers would be able to understand the markup better when they are looking at the Twig file.